templates/compose/calibre-web-automated.yaml

```
@@ -0,0 +1,19 @@
+# documentation: https://github.com/crocodilestick/Calibre-Web-Automated
+# slogan: Calibre-Web but automated and with Calibre features! Fully automate and simplify your eBook set up!
+# tags: calibre,calibre-web,ebook,library,epub,ereader,kindle,book,reader
+# logo: svgs/calibre-web-automated.svg
+# port: 8083
+services:
+  calibre-web-automated:
+    image: 'crocodilestick/calibre-web-automated:latest'
+    environment:
+      - SERVICE_FQDN_CWA_8083
```

new services now need to use the `SERVICE_URL` instead of the `SERVICE_FQDN` since the latest update from a few days ago.
```suggestion
      - SERVICE_URL_CWA_8083
```